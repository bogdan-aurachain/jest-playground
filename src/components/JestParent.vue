<template>
    <div class="bg-blue-1">
        <div class="col-12 full-width">PARENT</div>
        <div class="col-12 full-width q-ma-md bg-red-1"><jest-child/></div>
    </div>
</template>

<script>
import JestChild from "@/components/JestChild.vue";

export default {
    name: "JestParent",
    components: { JestChild }
};
</script>

<style scoped>

</style>