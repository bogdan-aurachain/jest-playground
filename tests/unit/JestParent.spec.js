import { mount } from "@vue/test-utils";
import JestParent from "@/components/JestParent.vue";

describe("testing Jest Parent component", () => {
    test("check html method", () => {
        const wrapper = mount(JestParent);
        expect(wrapper.element).toMatchSnapshot();
    });
});