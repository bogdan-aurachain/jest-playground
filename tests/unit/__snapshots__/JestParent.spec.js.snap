// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Jest Parent component check html method 1`] = `
<div
  class="bg-blue-1"
>
  <div
    class="col-12 full-width"
  >
    PARENT
  </div>
   
  <div
    class="col-12 full-width q-ma-md bg-red-1"
  >
    <div>
      <div>
        CHILD
      </div>
    </div>
  </div>
</div>
`;
